src/Symfony/Component/Lock/LockInterface.php

```diff
@@ -38,10 +38,12 @@ public function acquire($blocking = false);
     /**
      * Increase the duration of an acquired lock.
      *
+     * @param float|null $ttl Maximum expected lock duration in seconds
+     *
      * @throws LockConflictedException If the lock is acquired by someone else
      * @throws LockAcquiringException  If the lock can not be refreshed
      */
-    public function refresh();
+    public function refresh(/* $ttl = null */);
```

Comment:
I'm not such this is the right way to keep BC layer on interfaces. ping @nicolas-grekas 

Is there a way to trigger a deprecation?

Reply:
Maybe you can trigger a deprecation in the interface like https://github.com/symfony/symfony/commit/e8351d8f5465556c19280b98a61fe894a78d2c45#diff-7994cccf0d9c75a5a481e1240a159cce